README.md

```diff
@@ -28,6 +28,21 @@ what_is(Erlang) ->
 
 *Disclaimer: erlfmt is just a code formatter, not a solution to all life's problems.*
 
+## Comparison with other Erlang formatters
+
+|                               |erlfmt                                                           |rebar3_format                |steamroller                                    |erl_tidy |
+|---                            |---                                                              |---                          |---                                            |--- |
+|File Types                     |.erl, .hrl, .app, .app.src, .config, .script, .escript           |.erl, .hrl	                |.erl, .hrl, .app, .app.src, .config, .script	|.erl |
+|Macros                         |No crashes formatting OTP                                        |Skips entire files sometimes	|Skips entire files sometimes	                |Crashes sometimes |
+|Comments                       |Preserves and moves to line before                               |Preserves but Floating       |Crashes sometimes and Reorders	                |Crashes sometimes and Floating |
+|Configurable vs Opinionated    |Opinionated                                                      |Configurable                 |Opinionated                                    |Configurable |
+|Preserving Representation      |Yes                                                              |Some                         |No                                             |No |
+|Line Break Hints               |Yes                                                              |No                           |No                                             |No |
+|Opt In/Out                     |per file, per top level expression                               |per file                     |No                                             |No |
+|Speed                          |OTP lib in 7s                                                    |N/A                          |N/A                                            |N/A |
+
+See the [comparison with other erlang formatters document](./doc/ErlangFormatterComparison.md) for more details.
```

Comment:
I think we should move this section a bit below, especially below install/usage instructions. What do you think?

Perhaps we need a small ToC at the beginning of the readme?

Reply:
I think we should keep the references as close as possible to the content, but a TBC is a great idea.
I'll add one in a following pull request.

Reply:
Pull request for TBC https://github.com/WhatsApp/erlfmt/pull/132